dashboard/test/ui/features/platform/global_edition/region_select.feature

```diff
@@ -31,23 +31,23 @@ Feature: Global Edition - Region Select
   Scenario: User can switch between the international and regional versions using the language selector on a Lab page
     Given I am on "http://studio.code.org/projects/artist/new"
     And I wait for the lab page to fully load
-    And element "#locale option:checked" contains text "English"
+    And element "#localeForm option:checked" contains text "English"
```

Comment:
was the id meant to have changed to `localeForm`? it looks like there's no id instead

Reply:
https://github.com/code-dot-org/code-dot-org/blob/88877174f9451cdbcc4bb05f562ca56f6ee65c55/apps/src/sharedComponents/footer/I18nDropdown/index.tsx#L30

Reply:
https://github.com/code-dot-org/code-dot-org/blob/88877174f9451cdbcc4bb05f562ca56f6ee65c55/apps/src/sharedComponents/footer/I18nDropdown/index.tsx#L27-L30